server_py/flatgov/templates/bills/detail.html

```diff
@@ -60,142 +152,61 @@ <h4>Analyze Related Bills</h4>
             aria-controls="current" aria-selected="true">All</a>
         </li>
     </ul>
-    <div class="tab-content light-gray-tabs p-4" id="relatedBillsTableContent">
-        <div class="tab-pane fade show active" id="current" role="tabpanel" aria-labelledby="current-tab">            
-            <table class="table mt-3" id="related-bills-table">
+    <div class="tab-content light-gray-tabs p-4" id="crsReportContent">
+        <div class="tab-pane fade show active" id="crs-report" role="tabpanel" aria-labelledby="crs-report-tab">
+            <h6 class="pt-3">
+                View CRS reports for {{ bill.get_type_abbrev }} {{ bill.number }} in the {{ bill.congress }}th Congress
```

Comment:
CRS reports data in the wrong tab? This is related bills tab see comment in issue (#193)

It's better to not rush the fixes, you can make the fix but test it yourself otherwise it adds a lot of time to the cycle. In my experience, fixing bugs often causes more bugs. So it's really important to fully understand the context of the bug before and after, to test properly